pkg/summary/summarizers.go

```diff
@@ -50,6 +50,7 @@ var (
 		"ScalingActive":               "pending",
 		"AbleToScale":                 "pending",
 		"RunCompleted":                "running",
+		"Errored": "error",
```

Comment:
Can you run gofmt?  I'd consider a different name for the condition.  False means "bad", so Errored=False is bad and Errored=True is good.  Somewhat counter intuitive.